Swift 2.2

cocoahero opened the pull request:
- [x] Update project structure to new layout.
- [x] Use `jspahrsummers/xcconfigs` for base project / target build settings.
- [x] Fix new build errors.

cocoahero committed:
Switch to hyphenated scheme and target naming convention.

cocoahero committed:
Add xcconfigs submodule.

cocoahero committed:
Rename Configuration -> Platforms.

cocoahero committed:
Import xcconfigs into project.

cocoahero committed:
Set base configurations for targets.

cocoahero committed:
Fix broken Info.plist references.

cocoahero committed:
Remove redundant build settings.

cocoahero committed:
Fix type comparison error.

cocoahero committed:
Update travis configuration.

This temporarily uses xctool master in order to bypass
facebook/xctool#666.

calebd commented:
This looks good to me. Should we move the "Enable Testability" configuration from each target to the project? It would solve one of the project level warnings. Seems harmless enough. Otherwise this looks good.

cocoahero committed:
Switch to project level testability build setting.

cocoahero committed:
Add tvOS to Travis config.

cocoahero committed:
Switch to pod lib lint.

This checks the podspec in the current working directory, instead of
the trunk / remote version.
